core/src/main/java/bisq/core/api/CorePaymentAccountsService.java

```diff
@@ -173,6 +172,12 @@ void validateFormField(PaymentAccountForm form, PaymentAccountFormField.FieldId
         paymentAccount.validateFormField(form, fieldId, value);
     }
 
+    private void verifyAccountNameUniqueness(String accountName) {
+        if (getPaymentAccounts().stream().anyMatch(e -> e.getAccountName() != null &&
+                e.getAccountName().equals(accountName)))
+            throw new IllegalArgumentException(format("account %s is already taken", accountName));
```

Comment:
Please use the error message "Account '%s' is already taken" which is more suitable for the user.

Reply:
done